wgpu/src/util/init.rs

```diff
@@ -199,8 +199,9 @@ pub async fn is_browser_webgpu_supported() -> bool {
 /// this method will panic, see [`Instance::enabled_backend_features()`].
 #[allow(unused_mut)]
 pub async fn new_instance_with_webgpu_detection(
-    mut instance_desc: wgt::InstanceDescriptor,
+    instance_desc: impl std::ops::Deref<Target = wgt::InstanceDescriptor>,
```

Comment:
hum?

Reply:
urgh I kept that scribble by accident. Nothing to see here!